docs/tutorials/gpu.md

```diff
@@ -0,0 +1,83 @@
+# GPU simulations
+
+This tutorial demonstrates how to leverage GPUs to accelerate the simulation of quantum systems with dynamiqs. By directing our simulations to run on the GPU, we can significantly enhance the computational efficiency, especially for large-dimensional quantum systems.
+
+```python
+import torch
+import dynamiqs as dq
+```
+
+## The options dictionnary
+
+### Selecting a device
+
+Running simulations on a GPU is achieved in a single-line through the `options` argument of dynamiqs solvers.
+
+% skip: start
+
+```python
+# option 1: use a string
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+# option 2: use a torch.device
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+# pass the device to the options dictionnary and to your solver
+options = dict(device=device)
+result = dq.sesolve(..., options=options)
+```
+
+% skip: end
+
+Conversion of your tensors to the GPU is then handled automatically by dynamiqs. Note that the `device` argument is also available for the `mesolve` and `smesolve` solvers.
+
+!!! Warning "Time-dependent Hamiltonians"
```

Comment:
A good way of preventing this is to make sure that all objects involved in `H(t)` are already on GPU. If the users use `.to("cuda")` in `H(t)` there will be a huge penalty. It may be useful to explain it